plugins/module_utils/network/sonic/config/system/system.py

```diff
@@ -318,6 +322,10 @@ def get_create_system_request(self, want, commands):
         load_share_hash_algo_payload = self.build_create_load_share_hash_algo_payload(commands)
         if load_share_hash_algo_payload:
             request = {'path': load_share_hash_algo_path, 'method': method, 'data': load_share_hash_algo_payload}
+        audit_rules_path = 'data/openconfig-system:system/openconfig-system-ext:auditd-system/config/audit-rules'
+        audit_rules_payload = self.build_create_audit_rules_payload(commands)
+        if audit_rules_payload:
+            request = {'path': audit_rules_path, 'method': method, 'data': audit_rules_payload}
             requests.append(request)
```

Comment:
The auditd added instructions overwrote the "requests.append(request)" for "load_share_hash_algo".

Here is a recommended fix for that:

```suggestion
            requests.append(request)
        audit_rules_path = 'data/openconfig-system:system/openconfig-system-ext:auditd-system/config/audit-rules'
        audit_rules_payload = self.build_create_audit_rules_payload(commands)
        if audit_rules_payload:
            request = {'path': audit_rules_path, 'method': method, 'data': audit_rules_payload}
            requests.append(request)
```